pkgs/tools/security/clevis/default.nix

```diff
@@ -29,6 +30,11 @@ stdenv.mkDerivation rec {
     hash = "sha256-3J3ti/jRiv+p3eVvJD7u0ko28rPd8Gte0mCJaVaqyOs=";
   };
 
+  patches = [
+    # Replaces the clevis-decrypt 300s timeout to a 10s timeout
+    ./tang-timeout.patch
```

Comment:
**not a nit**: Ideally, this should be configurable, did you send it upstream?

Reply:
an alternative could be to write a `.curlrc` with all the timeouts you care about in the configuration directory. but I see the problem.